.dassie/config/initializers/riiif.rb

```diff
@@ -35,5 +29,5 @@
   Riiif.not_found_image = Rails.root.join('app', 'assets', 'images', 'us_404.svg')
   Riiif.unauthorized_image = Rails.root.join('app', 'assets', 'images', 'us_404.svg')
 
-  Riiif::Engine.config.cache_duration = 365.days
+  Riiif::Engine.config.cache_duration = 1.day
```

Comment:
What is the purpose of this change?

Reply:
For the dev environments the lower cache time will hopefully give any problems hidden by cached data a chance to surface. The generator for this file uses a month, which seems like a good default.